libgalois/src/analytics/k_truss/k_truss.cpp

```diff
@@ -217,20 +218,19 @@ struct KeepSupportedEdges {
 /// 3. Go back to 3.
 katana::Result<void>
 BSPTrussAlgo(Graph* g, unsigned int k) {
-  if (k - 2 == 0) {
+  if (k == 2) {
```

Comment:
@gurbinder533 Is this really correct? I have a strong feeling this should be `k < 2`.

Reply:
Yes, it should be k <= 2. It doesn't work for k == 2 as well. Not sure why it is just checking k == 2 in the old code :P